src/main/groovy/edu/oregonstate/mist/videogamesampleapi/core/ErrorPOJO.groovy

```diff
@@ -0,0 +1,11 @@
+package edu.oregonstate.mist.videogamesampleapi.core
+
+/**
+ * Created by george on 12/22/15.
+ * Error representation
+ */
+
+class ErrorPOJO {
```

Comment:
Instead of creating a new error class, this API should be able to use the Error class defined by the skeleton: https://github.com/osu-mist/web-api-skeleton/blob/master/src/main/groovy/edu/oregonstate/mist/api/Error.groovy